src/mono/wasm/runtime/exports.ts

```diff
@@ -115,23 +115,48 @@ export const BINDING: BINDING = <any>{
 // this is executed early during load of emscripten runtime
 // it exports methods to global objects MONO, BINDING and Module in backward compatible way
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-function export_to_emscripten(dotnet: any, mono: any, binding: any, internal: any, module: any): void {
+function exportAPI(mono: any, binding: any, internal: any, module: any, isGlobal: boolean, isNode: boolean, isShell: boolean, isWeb: boolean, glocateFile: Function): void {
```

Comment:
Can we make all these bools and glocateFile live in an object with keys?

Reply:
They are not bools, `mono` is seed object what would become `MONO` later. `isNode`, `locateFile` etc is import from emscripten which is not expoted on `Module` object. I will rename it to make it clearer.